src/endpoints/deleteUser.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from "express";
+
+import { AuthSchemaType } from "../utils/index.d";
+import userSchema from "../schema/authSchema";
+import debug from "../utils/debug";
+import messages from "../schema/messageSchema";
+
+/**
+ * This is the delete user endpoint. This endpoint will as the name suggests,
+ * Delete the provided user in the user field.
+ *
+ * @type {GET} This is a get typed endpoint.
+ * @param {string} user The user to delete.
+ */
+
+const deleteUser = async (req: Request, res: Response): Promise<void> => {
```

Comment:
Please add a next function

Reply:
Resolved in [7717d15](https://github.com/Chillandchat/api/pull/8/commits/7717d15a97872b27d6b59822cfbe1af0ccffaea3)